jobs/build/release/Jenkinsfile

```diff
@@ -101,6 +101,12 @@ node {
                         $class: 'BooleanParameterDefinition',
                         defaultValue: false
                     ],
+                    [
+                        name: 'OPEN_NEXT_RELEASE',
```

Comment:
on reflection, this seems wiser than having it always happen for every x86 release. the downside is that the human has to remember to click the box...

Reply:
on the other hand, it's going to be the exception that we don't want to open up for builds after x86. so maybe this flag should be `KEEP_RELEASE_CLOSED` and if not set, the next release is created (whenever x86 release is created, like cincinnati PRs).

Reply:
nice, inverted the flag here: https://github.com/openshift/aos-cd-jobs/pull/2307/commits/0413d3f62b4546134a103716cd4a83c7dadf76cf